### ISSUE TITLE
Unable to reproduce same accuracy reported - DCN_v2 (ResNet101 Backbone)

### ISSUE THREAD

**Issue by lzn87** [open-mmlab/mmdetection#4015]
Hi, I have been experimenting with deformable convolution on the COCO dataset. I used the following config:
`/configs/dcn/faster_rcnn_r101_fpn_dconv_c3-c5_1x_coco.py
`

After trained with 1 epoch, the model had accuracy on validation significantly lower than the accuracy reported in the corresponding log at [here](http://download.openmmlab.com/mmdetection/v2.0/dcn/faster_rcnn_r101_fpn_dconv_c3-c5_1x_coco/faster_rcnn_r101_fpn_dconv_c3-c5_1x_coco_20200203_230019.log.json). The following table summaries the difference:

Model | bbox_mAP | bbox_mAP_50 | bbox_mAP_75 | bbox_mAP_s | bbox_mAP_m | bbox_mAP_l
------------ | ------------- | ------------- | ------------- | ------------- | ------------- | -------------
model trained on my end| 0.122 | 0.243 | 0.11 | 0.064 | 0.13 | 0.163
pretrained model  | 0.242 | 0.434 | 0.247 | 0.124 | 0.274 | 0.325

I used the following command: 
`python -u tools/train.py configs/dcn/faster_rcnn_r101_fpn_dconv_c3-c5_1x_coco.py  --work-dir=. --launcher="slurm" 
`

For reference, my Python version was 3.7.3; PyTorch version was 1.5.0; Cuda: 10.1; TorchVision: 0.6.0a0+82fd1c8; MMCV: 1.1.5; MMdetection: 2.5.0+unknown; 8 GPUs were all RTX 2080Ti.

I noticed that my Pytorch and MMdetection versions were newer than the ones for the pre-trained model, could that be the reason? What can I do to reproduce the same results? Any help would be greatly appreciated!

**Comment by xvjiarui** [comment]
Hi @lzn87 
Did you modify any config or code? How many GPUs did you use for training?

**Comment by lzn87** [comment]
> Hi @lzn87
> Did you modify any config or code? How many GPUs did you use for training?

Hi,

Thanks for your reply, I didn't change config or any code. I used 8 GPUs for training. I guess it might be some backward-incompatible changes introduced in  v2.5.0 and v2.4.0 that caused this issue. I'm currently training with v2.3.0, but I don't know if this can help.

**Comment by xvjiarui** [comment]
Hi @lzhbrian 
Thanks for the report. We will look into it.

**Comment by AronLin** [comment]
I trained a model with this config and found that we can get normal results in our latest version.
